Add accordion to metadata form. (#5760)
plone/volto: docs/source/recipes/widget.md

```{note}
Please contribute to this section!
```

## Sidebar

> In the edit form a sidebar is used when the form contains block data. There are some helper actions methods which can be used to change the form state.

Reviewer: Avoid passive voice, prefer to address the reader. One sentence per line.

```suggestion
In the edit form, a sidebar is used when the form contains block data.
You can use the following helper action methods to change the form state.
```

### Setting tab

> You can use the `setSidebarTab` action to set the current active tab (metadata or block).

Reviewer: ```suggestion
You can use the `setSidebarTab` action to set the current active tab, either via metadata or a block.
```
